core/frontend/src/DisplayStyleState.ts

```diff
@@ -497,7 +505,7 @@ export abstract class DisplayStyleState extends ElementState implements DisplayS
   /** @internal */
   public async getMapLayerRange(layerIndex: number, isOverlay: boolean): Promise<MapCartoRectangle | undefined> {
     const mapLayerSettings = this.mapLayerAtIndex(layerIndex, isOverlay);
-    if (undefined === mapLayerSettings)
+    if (undefined === mapLayerSettings || mapLayerSettings instanceof ModelMapLayerSettings)
```

Comment:
getMapLayerRange is used by the 'zoom-to layer' comand in the UI (i.e DisplayStyleState.viewMapLayerRange).  Is there a way to make the view fit a model map layer?  Otherwise we will be forced to disable this feature for this type of layer

Reply:
Good point, I'll fix this...

Reply:
I've added implementation to return range for MMLs -- still need to test.